Web Site Pages/SRVvsIRV.txt

```diff
@@ -1,11 +1,13 @@
 <p><h3><strong>Traditional Instant Runoff or Version 2.0?</strong></h3>
 
+<h4>SRV is new IRV hybrid</h4>
+
 <div id="IRV"></div>
-Traditional Instant Runoff Voting (IRV) is the Ranked Choice O.G. - it's been used around the world and the states for more than a century. Score Runoff Voting (SRV) is a new kid on the block that uses an instant runoff to determine the final winner.
+Traditional Instant Runoff Voting (IRV) is the classic form of Ranked Choice — over more than a century, it's been used on and off in various places around the world. Score Runoff Voting (SRV) is a new hybrid version that combines score voting with an instant runoff to determine the final winner.
```

Comment:
I still think the O.G. bit isn't great, but please consider the other parts of this line's updates. I think the "over more than a century, it's been used on and off in various places around the world" reads really well to give an honest picture.